Package.swift

```diff
@@ -1,14 +1,30 @@
-//===----------------------------------------------------------------------===//
-//
-// This source file is part of the DeviceKit open source project
-//
-// Copyright © 2014 - 2018 Dennis Weissmann and the DeviceKit project authors
-//
-// License: https://github.com/dennisweissmann/DeviceKit/blob/master/LICENSE
-// Contributors: https://github.com/dennisweissmann/DeviceKit#contributors
-//
-//===----------------------------------------------------------------------===//
+// swift-tools-version:5.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription
 
-let package = Package(name: "DeviceKit")
+let package = Package(
+    name: "DeviceKit",
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "DeviceKit",
+            targets: ["DeviceKit"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages which this package depends on.
+        .target(
+            name: "DeviceKit",
+            dependencies: []),
+        .testTarget(
+            name: "DeviceKitTests",
+            dependencies: ["DeviceKit"],
+            path: "Tests"),
```

Comment:
Please remove the comma at the end of this line.
```suggestion
            path: "Tests")
```